src/wp-includes/global-styles-and-settings.php

```diff
@@ -92,6 +92,8 @@ function wp_get_global_settings( $path = array(), $context = array() ) {
  * Gets the styles resulting of merging core, theme, and user data.
  *
  * @since 5.9.0
+ * @since 6.3.0 the internal format "var:preset|color|secondary" is always resolved
+ *              to the standard form "var(--wp--preset--font-size--small)".
```

Comment:
Do we really need this comment given there are no actual changes to this function? I'm assuming that returning the internal notation was a bug 😄 as I don't see the behaviour documented anywhere.

Reply:
This is a hard one. I suppose the comment is helpful in clarifying the behavior so consumers using different versions of WordPress can prepare accordingly: they have to do the conversion themselves in versions <6.3. Unless this change is backported to any older version that is also affected :thinking: 

For the purposes of 6.3 beta, I'm going to merge this as it is, but we can iterate / create follow-ups after.